### DISCUSSION TITLE
I want to see who was the last one who have modified the string.

### DISCUSSION THREAD

**Discussion by stepanof** [nocodb/nocodb#2013] [Q&A] [answered]
Hello!

**Here is my issue**:
For example, I created a new project and added new table with 2 strings - String1 and String2.
Then I added Bob and Alice to this project as editors.
Then I want to see who modified String1 and String2 the last one (Bob or Alice)
I know that I can find this information in logs ([Audit](https://docs.nocodb.com/setup-and-usages/audit)) but it is not acceptable solution.

May be it is possible to add extra column "The last access" and automate filling this string from the rootDB? I suppose I can write a SQL script which will be able to give the email of editor who last modified each string (Bob or Alice).

I would appreciate any ideas how to implement this.

**Comment by o1lab** [comment, marked as answer]
You can expand that individual row and see all the edit history - who changed it to what.
<img width="1000" alt="Screenshot 2022-05-11 at 10 57 56" src="https://user-images.githubusercontent.com/5435402/167823982-d20e2a47-2d2b-4705-a32e-4a7bf8b7c2b2.png">

To answer the question otherwise : last_modified_by is not available at the moment. You can raise a feature request on this.

**Reply by stepanof** [reply]
Thank you for quick answer!
